app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
```

Comment:
[OPTIONAL] Kindly add MIT license to your project.

Reply:
Thanks for the feedback, I will do that